event/event.gen.go

```diff
@@ -1092,6 +1092,27 @@ func UserProximity(listener func(*vecty.Event)) *vecty.EventListener {
 	return &vecty.EventListener{Name: "userproximity", Listener: listener}
 }
 
+// A compatible VR device has been connected to the computer.
+//
+// https://developer.mozilla.org/docs/Web/Events/vrdisplayconnected
+func VRDisplayConnected(listener func(*vecty.Event)) *vecty.EventListener {
+	return &vecty.EventListener{Name: "vrdisplayconnected", Listener: listener}
+}
+
+// A compatible VR device has been disconnected from the computer.
+//
+// https://developer.mozilla.org/docs/Web/Events/vrdisplaydisconnected
+func VRDisplayDisconnected(listener func(*vecty.Event)) *vecty.EventListener {
+	return &vecty.EventListener{Name: "vrdisplaydisconnected", Listener: listener}
+}
+
+// The presenting state of a VR device has changed — i.e. from presenting to not presenting, or vice versa.
+//
+// https://developer.mozilla.org/docs/Web/Events/vrdisplaypresentchange
+func VRDisplayPresentChange(listener func(*vecty.Event)) *vecty.EventListener {
+	return &vecty.EventListener{Name: "vrdisplaypresentchange", Listener: listener}
+}
```

Comment:
FWIW, all these comments break the style rule that documentation should begin with the name of the identifier, e.g.:

```Go
// VRDisplayPresentChange ...
func VRDisplayPresentChange() {
```

And it's something `go lint` detects.

Since this documentation is generated from an upstream source that doesn't follow that rule, I'm not sure if anything can be done about it.